### ISSUE TITLE
more "shows" from Heise

### ISSUE THREAD

**Issue by GETandSELECT** [oscaropenness/plugin.video.heiseshowrss#2]
thanks for your work.

please add:

- Kurz informiert – die wichtigsten IT-News in 2 Minuten
- c't nachgehakt: Autoren über ihre Artikel

into Addons.

Is this possible?

**Comment by oscaropenness** [comment]
_Kurz informiert_ should be easy to do since there is an RSS-feed. I might include it in the next release of _#heiseshow_.
_nachgehakt_ will not work since the clips are encrypted and I cannot find them anywhere else.

But I will consider to aggregate the three video podcast into one _heise podcast_ plugin to keep everything in one place. I was already hesitant to have both _#heiseshow_ and _c't uplink_ seperate.

Aggregation will take a bit of time cannot tell, when I will find time.

**Comment by GETandSELECT** [comment]
You get the video streams direct from Heise servers?
On Youtube (warning Google) "nachgehakt" are without DRM.

**Comment by oscaropenness** [comment]
> You get the video streams direct from Heise servers?

Yes & no, they are stored on some cdn. But I parse the links from their RSS feed directly. But for _nachgehakt_ there is no RSS feed and I would have to scrape their website, and there I cannot see the actual URL.

> On Youtube (warning Google) "nachgehakt" are without DRM.

Correct, but the whole purpose of these two addons (#heiseshow & c't uplink) was to **avoid the Youtube plugin** (that's why they are named plugin.video.heiseshow**rss**).

The Youtube plugin has gotten [a bit of a b**ch to setup](https://github.com/jdf76/plugin.video.youtube/wiki/Personal-API-Keys) since [google shutdown the api-keys](https://github.com/jdf76/plugin.video.youtube/issues/736) of the developer.

If you actually **have the youtube plugin working** you can make your live quite easy just using the official heise channel there. You can use the [.STRM file option](https://kodi.wiki/view/Internet_video_and_audio_streams) with the link `plugin://plugin.video.youtube/play/?playlist_id=PLUoWfXKEShjeEJJ83lJkme2DKyFUrDBpz&order=default` for the [Youtube-Plugin](https://kodi.wiki/view/Add-on:YouTube#Integration_with_STRM_files) or just bookmark the channel in the plugin.

**Comment by GETandSELECT** [comment]
thanks for explanations.
No, I don't have a Google account and prefer Non-Google hosted stuff.

**Comment by GETandSELECT** [comment]
Hi @oscaropenness 

I found Mac and I podcast.

Is it possible to also add a plugin for https://www.heise.de/mac-and-i/mac-and-i-podcast.rss?

[More info:](https://www.heise.de/mac-and-i/meldung/Die-eigene-Cloud-im-Podcast-von-Mac-i-4781286.html?wt_mc=nl.red.mac-and-i.mac-and-i-nl.2020-06-12.link.link)

> Der Apple-Podcast von Mac & i erscheint monatlich
> 
> Den Apple-Podcast von Mac & i können Sie per RSS-Feed (Audio oder Video) mit der Podcast-App Ihrer Wahl abonnieren. Sie finden ihn zum Ansehen oder Anhören auch in Apples Podcast-Verzeichnis (Audio oder Video), bei YouTube und Spotify.

thanks
